src/main/java/ch/uzh/ifi/hase/soprafs22/service/LobbyService.java

```diff
@@ -83,7 +83,7 @@ public ILobby createLobby(String token, String lobbyName, Visibility visibility,
         }
 
         // Check if gameType equals ranked and the request is form an unregistered user
-        if(gameType.equals(GameType.RANKED) && registeredUser == null){
+        if (gameType.equals(GameType.RANKED) && registeredUser == null) {
```

Comment:
https://stackoverflow.com/questions/1750435/comparing-java-enum-members-or-equals

== instead of equals

Reply:
ahh good to know, will change it to ==

Reply:
As the entry you mentioned says, both are equally correct.

Reply:
The one with second most upvotes is actually a bit more precise and mentions that == has some benefits

- It works.
- It's faster.
- It's safer at run-time.
- It's safer at compile-time.

I think the danger of equals is mostly due to null pointer exception, which is no problem for ==

Reply:
changed it to ==

Reply:
cool